### PR TITLE
Update angularfire override with version 1.1.4

### PR DESCRIPTION
**Pull request by dinoboff**

### CODE REVIEW

**Comment by guybedford** [comment]
Thanks! Unfortunately because we missed the major release here already long ago, this will be a breaking change for anyone using it already with custom shims. We should remove the directories.lib override to ensure it is non-breaking.

**Comment by dinoboff** [comment]
@guybedford removed

File: package-overrides/github/firebase/angularfire@1.1.4.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "main": "dist/angularfire.js",
+  "shim": {
+    "*": {
```
Comment (guybedford) [inline package-overrides/github/firebase/angularfire@1.1.4.json]: This should just be `"dist/angularfire"` I think?

**Comment by guybedford** [comment]
Thanks please let me know when you've updated again re the comment above.

**Comment by dinoboff** [comment]
@guybedford updated

**Comment by guybedford** [comment]
:+1:

**Pull request merged**